refactor nextSectionRef

adeds opened the pull request.

adeds committed:
refactor(data, domain, DynamicUI, myJobs) : refactor nextSectionRef structure

mochadwi commented:
The builds failing, see here for more details: https://github.com/sampingantech/dynamic-ui/pull/7/checks?check_run_id=1253036236#step:5:10744

You just needs to increment `versionName` in `root/build.gradle` @adeds

mochadwi committed:
build(version): Bump version to 0.10.4

1. Move out logic jump properties (`nextSectionRef`) as sibling `properties` instead of nested

mochadwi committed:
Merge branch 'develop' into refactor

adesamp commented:
> The builds failing, see here for more details: https://github.com/sampingantech/dynamic-ui/pull/7/checks?check_run_id=1253036236#step:5:10744
> 
> You just needs to increment `versionName` in `root/build.gradle` @adeds

ohh i see, thanks a lot @mochadwi 🙏
